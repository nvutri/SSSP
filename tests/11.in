c random graph n=32 m=128 lo=0 hi=4194304 seed=1292225576
p sp 32 128
a 15 19 2987137
a 19 8 321330
a 8 9 372649
a 9 13 2958019
a 13 21 4022137
a 21 18 2445564
a 18 16 1119677
a 16 2 130825
a 2 20 2628981
a 20 4 660665
a 4 29 1645611
a 29 3 3546361
a 3 26 76468
a 26 28 221277
a 28 22 2554189
a 22 32 1674132
a 32 25 363854
a 25 6 2022496
a 6 30 1272490
a 30 11 2963089
a 11 14 1321304
a 14 5 2058897
a 5 1 972634
a 1 12 3173238
a 12 27 3584076
a 27 24 289204
a 24 7 1523710
a 7 10 3877639
a 10 17 2502989
a 17 23 672270
a 23 31 4033482
a 31 15 633966
a 14 1 2278036
a 2 9 3690750
a 1 22 1643879
a 10 17 1594383
a 20 13 3091819
a 24 25 3551849
a 18 17 2794425
a 18 22 3307502
a 8 13 3474111
a 32 11 2717669
a 24 26 2220431
a 20 12 1403639
a 8 13 2905055
a 26 4 619166
a 27 17 2140162
a 22 2 1680781
a 30 15 105671
a 13 19 395672
a 14 25 1784178
a 6 16 3811871
a 14 8 1700873
a 24 29 2375434
a 21 14 1418322
a 19 14 3958821
a 10 30 3238438
a 16 24 1704248
a 14 25 1829744
a 23 26 3604954
a 29 28 1292817
a 11 32 4170366
a 32 16 294487
a 21 29 830535
a 3 12 2008746
a 8 13 3034756
a 9 20 3122159
a 31 20 968811
a 1 31 1386288
a 3 23 4074815
a 24 25 2949513
a 8 22 3266597
a 20 5 1267967
a 11 2 3313820
a 12 23 649350
a 21 11 3105714
a 1 8 2596361
a 1 25 215467
a 8 22 358940
a 3 2 588285
a 3 32 2641185
a 30 25 449089
a 28 16 1182318
a 23 1 1035275
a 14 32 3893668
a 2 11 3118276
a 4 2 744864
a 26 4 3268878
a 25 18 3669941
a 2 21 2431443
a 1 9 3711718
a 8 27 152906
a 1 27 4115181
a 3 29 247142
a 20 6 526531
a 14 30 2075743
a 13 19 1706859
a 1 5 3164950
a 28 17 4029841
a 2 24 2115668
a 6 20 2916325
a 13 19 729044
a 29 3 2753024
a 1 15 109117
a 27 8 2165042
a 27 4 3922348
a 13 3 839353
a 7 25 3195618
a 26 17 3183617
a 20 19 3910284
a 20 30 93039
a 12 10 924925
a 14 13 3794400
a 30 15 761302
a 6 8 2927490
a 30 9 19737
a 21 18 465227
a 21 9 145630
a 15 32 816859
a 31 23 464688
a 23 10 1413939
a 30 23 3266023
a 14 9 1281411
a 3 10 3126534
a 29 5 253762
a 22 26 2596824
a 8 6 1207181
a 9 30 906445
