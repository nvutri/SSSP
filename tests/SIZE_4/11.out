1 0
2 1.35129e+06
3 6.26525e+06
4 3.01576e+06
5 3.16495e+06
6 2.23796e+06
7 4.44842e+06
8 2.59636e+06
9 2.46788e+06
10 3.26492e+06
11 3.64364e+06
12 3.17324e+06
13 5.42589e+06
14 4.96495e+06
15 109117
16 1.22046e+06
17 4.8593e+06
18 3.88541e+06
19 3.09625e+06
20 2.3551e+06
21 3.78273e+06
22 1.64388e+06
23 1.85098e+06
24 2.92471e+06
25 215467
26 4.2407e+06
27 2.74927e+06
28 4.46198e+06
29 4.61327e+06
30 2.44814e+06
31 1.38629e+06
32 925976
